autoscheduler/scraper/tests/models_tests.py

```diff
@@ -1,10 +1,13 @@
 """ Tests all of the models functions """
 
 import unittest
+import django.test
 
 from scraper.models.course import generate_course_id
 from scraper.models.department import generate_department_id
-from scraper.models.section import generate_meeting_id
+from scraper.models.section import generate_meeting_id, Section
+from scraper.models.grades import Grades
+from scraper.models.instructor import Instructor
 
 # A lot of these funcitons have to be self in order to be part of the TestCase and thus,

```

Comment:
```suggestion
# A lot of these functions have to be self in order to be part of the TestCase and thus,

```
I know you didn't actually change this but we should fix this typo anyways

Reply:
So we can actually remove these comment entirely, as well as the `#pylint: disable=no-self-use`, since we're using `self.assertCondition` instead of `assert condition` now